LICENSE

```diff
@@ -1,3 +1,10 @@
+The source code is licenced under Appache Licence Version 2.0 (see below).
```

Comment:
Yeah we may want to keep LICENSE unmodified, include the CC license in a different file, and describe the details in Credits.md (or a new file). I just noticed yesterday that GitHub now even auto-detects the license and shows it in the little repo details bar, maybe a modified file would cause trouble there?